ios/TextilePhotos.xcodeproj/project.pbxproj

```diff
@@ -1984,10 +1984,10 @@
 					"-ObjC",
 					"-lc++",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = io.textile.Wallet;
+				PRODUCT_BUNDLE_IDENTIFIER = io.textile.WalletDebug;
 				PRODUCT_NAME = TextilePhotos;
 				PROVISIONING_PROFILE = "2db150d1-101b-4ac0-b886-3e69f53beb8a";
-				PROVISIONING_PROFILE_SPECIFIER = "match Development io.textile.Wallet";
+				PROVISIONING_PROFILE_SPECIFIER = "match Development io.textile.WalletDebug";
```

Comment:
If I understand this correctly, it changes the bundle id and provisioning profile for the main application target... So what happens when we do a release build? How do we use to real bundle id and provisioning profile?

Reply:
Well, I'm changing the bundle id on Debug builds so that it is a duplicate app on your phone. So to do that, I needed a signing cert that was able to sign that new bundle id. Bundle id for the release build is still our original and uses the same signing cert we had before,

![screen shot 2018-09-18 at 8 16 19 am](https://user-images.githubusercontent.com/370259/45697738-4de27d00-bb1b-11e8-87bf-d594ea43805a.png)

Reply:
Not sure, I think that's what that SO is all about. Trying to learn more about the issue now...

Reply:
okay, sweet. let me know what you want to do